Makefile

```diff
@@ -23,7 +23,7 @@ VERSION ?= v$(VERSION_MAJOR).$(VERSION_MINOR).$(VERSION_BUILD)
 DEB_VERSION ?= $(VERSION_MAJOR).$(VERSION_MINOR).$(VERSION_BUILD)
 RPM_VERSION ?= $(VERSION_MAJOR).$(VERSION_MINOR).$(VERSION_BUILD)
 # used by hack/jenkins/release_build_and_upload.sh and KVM_BUILD_IMAGE, see also BUILD_IMAGE below
-GO_VERSION ?= 1.12.8
```

Comment:
we can't use version 1.12.9 yet since the kube-cross image is not available in that version yet.

Reply:
k8s.gcr.io/kube-cross:v1.12.9-1 has been released 3 days ago, and upstream K/K repo has switched to this version.

https://console.cloud.google.com/gcr/images/google-containers/GLOBAL/kube-cross?gcrImageListsize=30

Reply:
Image downloaded fine here.

Reply:
ok ! then I am fine with this change then